wire-ios/Wire-iOS/Sources/Components/AccentColorChangeHandler.swift

```diff
@@ -36,7 +36,7 @@ final class AccentColorChangeHandler: NSObject, ZMUserObserver {
         handlerBlock = changeHandler
         self.observer = observer
 
-        if let selfUser = SelfUser.provider?.selfUser, let userSession = ZMUserSession.shared() {
+        if let selfUser = SelfUser.provider?.providedSelfUser, let userSession = ZMUserSession.shared() {
```

Comment:
I guess the ZMUserSession.shared() is for later right?

Reply:
Yeah... we can't deal with all `ZMUserSession.shared()` in one PR, so each PR will deal with a bunch of them until we have none left.